src/components/pages/PostPage.tsx

```diff
@@ -99,6 +99,8 @@ class PostPage extends React.Component<Props, State> {
     const { sideContent, currentPage, parentPage, pages, leftMenuCurrentTopPage } = this.state;
     const loactionPathnameArrayRaw = (locationPath ? locationPath.replace(/\//g, " ") || "" : "").split(" ");
     const loacationPathnameArray = loactionPathnameArrayRaw.splice(1, (loactionPathnameArrayRaw.length -1 ) - 1);
+    const checkContent = React.Children.map(sideContent, child => child ? child.props.children.length:0);
```

Comment:
```suggestion
    const checkContent = React.Children.map(sideContent, child => child ? child.props.children.length : 0);
```

Reply:
If it is x ? a : b function then spaces are missing

Reply:
Theres an "if" because there's possibility that child.props.childern is undefined, so it has to be tested. In that case result is 0 and right block gets display: none.